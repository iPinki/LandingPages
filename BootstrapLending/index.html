<!DOCTYPE html>
<html lang="en" xmlns="http://www.w3.org/1999/html">
<head>
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css">
    <link rel="stylesheet" href="css/style.css">
    <meta charset="UTF-8">
    <title>Title</title>
</head>
<body>
<!--NAV-->
    <nav class="navbar navbar-expand-md navbar-light bg-dark sticky-top">
        <div class="container-fluid">
            <a href="#" class="navbar-brand"><img src="img/logo.png" alt=""></a>
            <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarResponsive">
                <span class="navbar-toggler-icon"></span>
            </button>
            <div class="collapse navbar-collapse" id="navbarResponsive">
                <ul class="navbar-nav ml-auto">
                    <li class="nav-item active">
                        <a href="#" class="nav-link">Главная</a>
                    </li>
                    <li class="nav-item">
                        <a href="#111" class="nav-link">Создание сайтов</a>
                    </li>
                    <li class="nav-item">
                        <a href="#222" class="nav-link">Уроки</a>
                    </li>
                    <li class="nav-item">
                        <a href="#333" class="nav-link">Контакты</a>
                    </li>
                </ul>
            </div>
        </div>
    </nav>
<!--/NAV-->

<!--SLIDER-->
    <div class="carousel slide" data-ride="carousel" id="slides">
        <ul class="carousel-indicators">
            <li data-target="#slides" data-slide-to="0" class="active"></li>
            <li data-target="#slides" data-slide-to="1" ></li>
            <li data-target="#slides" data-slide-to="2" ></li>
            <li data-target="#slides" data-slide-to="3" ></li>
            <li data-target="#slides" data-slide-to="4" ></li>
            <li data-target="#slides" data-slide-to="5" ></li>
        </ul>
        <div class="carousel-inner">
            <div class="carousel-item active">
                <img src="img/slider/1.jpg" alt="">
                <div class="carousel-caption">
                    <h1 class="display-2">proglib</h1>
                    <h3>Научитесь программировать</h3>
                    <button type="button" class="btn btn-outline-light btn-lg">Посмотреть</button>
                    <button type="button" class="btn btn-warning btn-lg">Демо сайта</button>
                </div>
            </div>
            <div class="carousel-item ">
                <img src="img/slider/2.jpg" alt="">
            </div>
            <div class="carousel-item ">
                <img src="img/slider/3.jpg" alt="">
            </div>
            <div class="carousel-item ">
                <img src="img/slider/4.jpg" alt="">
            </div>
            <div class="carousel-item ">
                <img src="img/slider/5.jpg" alt="">
            </div>
            <div class="carousel-item ">
                <img src="img/slider/6.jpg" alt="">
            </div>
        </div>
    </div>
<!--/SLIDER-->

<!--CONTAINER-->
    <div class="container-fluid">
        <div class="row jumbotron">
            <div class="col-xs-12 col-sm-12 col-md-9 col-lg-9 col-xl-10">
                <p class="lead">Lorem ipsum dolor sit amet, consectetur adipisicing elit.
                    Aliquam aut debitis, dolorum fugiat hic itaque magnam maxime veniam.
                    Amet aperiam consequuntur cumque impedit incidunt modi odit optio sequi totam, ut.
                Lorem ipsum dolor sit amet, consectetur adipisicing elit. Exercitationem laboriosam odit quisquam sapiente ullam voluptatem?</p>
            </div>
            <div class="col-xs-12 col-sm-12 col-md-3 col-lg-3 col-xl-2">
                <a href="#"><button class="btn btn-success btn-lg" type="button">Читать далее</button></a>
            </div>
        </div>
    </div>
<!--/CONTAINER-->

<!--CHARACTERISTICS-->
    <div class="container-fluid">
        <div class="row text-center alert">
            <div class="col-12">
                <h1 class="display-4" id="111">Создание сайтов</h1>
            </div>
            <hr>
            <div class="col-12">
                <p class="lead">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Adipisci alias atque
                    deleniti dicta, dignissimos dolores ea earum eligendi error eum excepturi exercitationem expedita harum inventore ipsum laborum modi
                    molestiae nobis non nulla obcaecati pariatur, perspiciatis quae quam quo repellendus sed sint tenetur
                    voluptate voluptatem! Architecto aut laborum quasi tempora voluptas.</p>
            </div>
        </div>
    </div>
<!--/CHARACTERISTICS-->

<!--LANGUAGE DESCRIPTION-->
    <div class="container-fluid padding">
        <div class="row text-center padding">
            <div class="col-xs-12 col-sm-6 col-md-4">
                <i class="fas fa-code"></i>
                <h3>HTML5</h3>
                <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Amet, voluptates!</p>
            </div>
            <div class="col-xs-12 col-sm-6 col-md-4">
                <i class="fas fa-bold"></i>
                <h3>Bootstrap</h3>
                <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Amet, voluptates!</p>
            </div>
            <div class="col-xs-12 col-sm-6 col-md-4">
                <i class="fab fa-css3"></i>
                <h3>CSS3</h3>
                <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Amet, voluptates!</p>
            </div>
        </div>
        <hr class="my-4">
    </div>
<!--/LANGUAGE DESCRIPTION-->

<!--IMAGE SECTION-->
    <div class="container-fluid padding">
        <div class="row padding">
            <div class="col-lg-6">
                <h2 id="222">Что-то про программирование</h2>
                <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. 
                    Aliquid dicta dignissimos ducimus eos excepturi facilis fuga ipsam iste nam nobis non odio placeat, 
                    provident quidem saepe sint tenetur unde voluptas?
                    <br><br>
                    Lorem ipsum dolor sit amet, consectetur adipisicing elit.
                    Aliquid dicta dignissimos ducimus eos excepturi facilis fuga ipsam iste nam nobis non odio placeat,
                    provident quidem saepe sint tenetur unde voluptas?
                    <br><br>
                    Lorem ipsum dolor sit amet, consectetur adipisicing elit.
                    Aliquid dicta dignissimos ducimus eos excepturi facilis fuga ipsam iste nam nobis non odio placeat,
                    provident quidem saepe sint tenetur unde voluptas?</p>
                <br>
                <a href="#" class="btn btn-danger">Читать далее</a>
            </div>
            <div class="col-lg-6">
                <img src="https://leonardo.osnova.io/8b115abf-1607-5dc1-95f3-be3a31296a12/-/scale_crop/900x461/center/-/format/webp/" alt="" class="img-fluid">
            </div>
        </div>
    </div>
<!--/IMAGE SECTION-->

<!--FIXED IMAGE SECTION-->
    <figure>
        <div class="fix-wrap">
            <div id="fixed">
                
            </div>
        </div>
    </figure>
<!--/FIXED IMAGE SECTION-->

<!--CLOSED SECTION-->
    <div class="text-center">
        <button type="button" id="gifsBtn" class="btn btn-success text-center" data-toggle="collapse"
                data-target="#gifs">Открыть блок</button>
    </div>
    <div id="gifs" class="collapse">
        <div class="container-fluid padding">
            <div class="row text-center">
                <div class="col-sm-6 col-md-3">
                    <img src="img/gif/1.gif" alt="" class="gif">
                </div>
                <div class="col-sm-6 col-md-3">
                    <img src="img/gif/2.gif" alt="" class="gif">
                </div>
                <div class="col-sm-6 col-md-3">
                    <img src="img/gif/3.gif" alt="" class="gif">
                </div>
                <div class="col-sm-6 col-md-3">
                    <img src="img/gif/4.gif" alt="" class="gif">
                </div>
            </div>
        </div>
    </div>
<!--/CLOSED SECTION-->

<!--PROG LANG CARDS-->
    <div class="container-fluid padding">
        <div class="row alert text-center">
            <div class="col-12">
                <h1 class="display-4">Языки программирования</h1>
            </div>
            <hr>
        </div>
    </div>
    <div class="container padding">
        <div class="row padding">
            <div class="col-md-4">
                <div class="card">
                    <img src="https://itproger.com/img/courses/1476977240.jpg" alt="" class="card-img-top">
                    <div class="card-body">
                        <h4 class="card-title text-center">HTML5</h4>
                        <p class="card-text">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Alias deserunt porro reiciendis!</p>
                        <a href="#" class="btn btn-warning">Посмотреть</a>
                    </div>
                </div>
            </div>
            <div class="col-md-4">
                 <div class="card">
                    <img src="https://itproger.com/img/courses/1551371255.jpg" alt="" class="card-img-top">
                    <div class="card-body">
                        <h4 class="card-title text-center">ASP.NET Core MVC / С#</h4>
                        <p class="card-text">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Alias deserunt porro reiciendis!</p>
                        <a href="#" class="btn btn-warning">Посмотреть</a>
                    </div>
                </div>
            </div>
            <div class="col-md-4">
                <div class="card">
                    <img src="https://itproger.com/img/courses/1508841661.jpg" alt="" class="card-img-top">
                    <div class="card-body">
                      <h4 class="card-title text-center">Python, Django</h4>
                        <p class="card-text">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Alias deserunt porro reiciendis!</p>
                        <a href="#" class="btn btn-warning">Посмотреть</a>
                    </div>
                </div>
            </div>
        </div>
    </div>
<!--/PROG LANG CARDS-->

<!--CONTACTS-->
    <div class="container-fluid padding">
        <section id="contact">
        <h2 class="mb-5 font-weight-bold text-center" id="333">Свяжитесь с нами</h2>
        <div class="row">
            <div class="col-lg-5 col-md-12">
                <form class="p-5">
                    <div class="md-form form-sm">
                        <i class="fa fa-user prefix grey-text"></i>
                        <input type="text" id="form3" class="form-control form-control-sm">
                        <label for="form3">Ваше Имя</label>
                    </div>
                    <div class="md-form form-sm">
                        <i class="fa fa-envelope prefix grey-text"></i>
                        <input type="text" id="form2" class="form-control form-control-sm">
                        <label for="form2">Ваша Почта</label>
                    </div>
                    <div class="md-form form-sm">
                        <i class="fa fa-tag prefix grey-text"></i>
                        <input type="text" id="form1" class="form-control form-control-sm">
                        <label for="form3">Тема</label>
                    </div>
                    <div class="md-form form-sm">
                        <i class="fa fa-pencil prefix grey-text"></i>
                        <textarea type="text" id="form0" class="md-textarea form-control form-control-sm" rows="3"></textarea>
                        <label for="form0">Ваше Письмо</label>
                    </div>
                    <div class="text-center mt-4">
                        <button class="btn btn-primary waves-effect waves-light">Отправить
                            <i class="fa fa-location ml-1"></i></button>
                    </div>
                </form>
            </div>
            <div class="col-lg-7 col-mg-12">
                <div class="row text-center">
                    <div class="col-lg-4 col-md-12 mb-3">
                        <p><i class="fa fa-map fa-1x mr-2 grey-text"></i>Киев</p>
                    </div>
                    <div class="col-lg-4 col-md-6 mb-3">
                        <p><i class="fa fa-building fa-1x mr-2 grey-text"></i>Понедельник - Пятница</p>
                    </div>
                    <div class="col-lg-4 col-md-6 mb-3">
                        <p><i class="fa fa-phone fa-1x mr-2 grey-text"></i>+380 (12) 345 67 89</p>
                    </div>
                </div>

                <div id="map-container-google-1" class="z-depth-1-half map-container" style="height: 500px">
                    <iframe src="https://maps.google.com/maps?q=kiev&t=&z=13&ie=UTF8&iwloc=&output=embed" frameborder="0"
                            style="border:0" allowfullscreen></iframe>
                </div>

            </div>
        </div>
        </section>
    </div>
<!--/CONTACTS-->

<!--FOOTER-->
    <footer class="container-fluid">
        <div class="container-fluid">
            <div class="row padding text-center">
                <div class="col-12">
                    <h2>Наши контакты</h2>
                </div>
                <div class="col-12 social padding">
                    <a href="#"><i class="fab fa-twitter"></i></a>
                    <a href="#"><i class="fab fa-instagram"></i></a>
                    <a href="#"><i class="fab fa-youtube"></i></a>
                    <a href="#"><i class="fab fa-facebook"></i></a>
                    <a href="#"><i class="fab fa-google-plus-g"></i></a>
                </div>
            </div>
        </div>
    </footer>
<!--/FOOTER-->


<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js"></script>
<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js"></script>
<script src="https://use.fontawesome.com/releases/v5.0.8/js/all.js"></script>
</body>
</html>