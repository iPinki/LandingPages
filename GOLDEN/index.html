<!DOCTYPE html>
<html lang="en">
<head>
    <link rel="stylesheet" href="css/style.css">
    <meta charset="UTF-8">
    <title>Title</title>
</head>
<body>

<div id="wrapper">
    <div id="header">
        <div class="container">
            <div class="logo">
                <a href=""><img src="img/logo.png" alt=""></a>
            </div>

            <ul class="nav">
                <li><a href="#">Home</a></li>
                <li><a href="#">Services</a></li>
                <li><a href="#">Portfolio</a></li>
                <li><a href="#">About</a></li>
                <li><a href="#">Contact</a></li>
            </ul>
        </div>
    </div>

    <div id="home">
        <div class="container">
            <h3>Welcome To Our Studio!</h3>
            <h1>It`s nice to meet you</h1>
            <button class="btn">Tell me more</button>
        </div>
    </div>

    <div id="services" class="box">
        <div class="container">
            <h2 class="title">Services</h2>
            <p class="descriptions">Proin iaculis purus consequat sem cure.</p>

            <div class="services-wrap clearfix">
                <div class="services-item commerce">
                    <h5 class="title">E-Commerce</h5>
                    <p>Proin iaculis purus consequat sem cure
                        digni ssim. Donec porttitora entum suscipit
                    </p>
                </div>
                <div class="services-item responsive">
                    <h5 class="title">Responsive Web</h5>
                    <p>Proin iaculis purus consequat sem cure
                        digni ssim. Donec porttitora entum suscipit
                    </p>
                </div>
                <div class="services-item security">
                    <h5 class="title">Web Security</h5>
                    <p>Proin iaculis purus consequat sem cure
                        digni ssim. Donec porttitora entum suscipit
                    </p>
                </div>
            </div>
        </div>
    </div>

    <div id="portfolio" class="box">
        <div class="container">
            <h2 class="title">Our portfolio</h2>
            <p class="descriptions">Proin iaculis purus consequat sem cure.</p>

            <div class="portfolio-wrap clearfix">
                <div class="portfolio-item">
                    <a href="#"><img src="img/portfolio/1.jpg" alt=""></a>
                    <div class="p-item-footer">
                        <h5 class="title">Ebony & Ivory</h5>
                        <span>Branding</span>
                    </div>
                </div>


                <div class="portfolio-item">
                    <a href="#"><img src="img/portfolio/2.jpg" alt=""></a>
                    <div class="p-item-footer">
                        <h5 class="title">Smart Stationary</h5>
                        <span>Print Design</span>
                    </div>
                </div>


                <div class="portfolio-item">
                    <a href="#"><img src="img/portfolio/3.jpg" alt=""></a>
                    <div class="p-item-footer">
                        <h5 class="title">Clever Poster</h5>
                        <span>Print Design</span>
                    </div>
                </div>


                <div class="portfolio-item">
                    <a href="#"><img src="img/portfolio/4.jpg" alt=""></a>
                    <div class="p-item-footer">
                        <h5 class="title">Vinyl Record</h5>
                        <span>Product Mock-Up</span>
                    </div>
                </div>


                <div class="portfolio-item">
                    <a href="#"><img src="img/portfolio/5.jpg" alt=""></a>
                    <div class="p-item-footer">
                        <h5 class="title">Treehouse Template</h5>
                        <span>Web Design</span>
                    </div>
                </div>


                <div class="portfolio-item">
                    <a href="#"><img src="img/portfolio/6.jpg" alt=""></a>
                    <div class="p-item-footer">
                        <h5 class="title">Burned Logo</h5>
                        <span>Branding</span>
                    </div>
                </div>
            </div>
        </div>
    </div>
</div>

<div id="about" class="box">
    <div class="container">
        <h2 class="title">About us</h2>
        <p class="descriptions">Proin iaculis purus consequat sem cure.</p>

        <div class="about-wrap">
            <div class="about-item clearfix">
                <div class="about-text left text-right">
                    <h5 class="title">JULI 2010<br> Our Humble Beginnings</h5>
                    <p>Lorem ipsum dolor sit amet, consectetur<br>
                        adipisicing elit. Aliquid aut autem, beatae<br>
                        dicta dignissimos dolorem est ex excepturi facilis<br>
                        ipsam itaque nostrum nulla odio repellendus repudiandae
                        sed unde vero voluptatem?</p>
                </div>
                <div class="about-img left">
                    <img src="img/AboutUs/1.jpg" alt="">
                </div>
            </div>

            <div class="about-item clearfix">
                <div class="about-text right text-left">
                    <h5 class="title">JANUARY 2011<br> Facing Startup Buttles</h5>
                    <p>Lorem ipsum dolor sit amet, consectetur<br>
                        adipisicing elit. Aliquid aut autem, beatae<br>
                        dicta dignissimos dolorem est ex excepturi facilis<br>
                        ipsam itaque nostrum nulla odio repellendus repudiandae
                        sed unde vero voluptatem?</p>
                </div>
                <div class="about-img right">
                    <img src="img/AboutUs/2.jpg" alt="">
                </div>
            </div>

            <div class="about-item clearfix">
                <div class="about-text left text-right">
                    <h5 class="title">DECEMBER 2012<br> Enter The Dark Days</h5>
                    <p>Lorem ipsum dolor sit amet, consectetur<br>
                        adipisicing elit. Aliquid aut autem, beatae<br>
                        dicta dignissimos dolorem est ex excepturi facilis<br>
                        ipsam itaque nostrum nulla odio repellendus repudiandae
                        sed unde vero voluptatem?</p>
                </div>
                <div class="about-img left">
                    <img src="img/AboutUs/3.jpg" alt="">
                </div>
            </div>

            <div class="about-item clearfix">
                <div class="about-text right text-left">
                    <h5 class="title">FEBRUARY 2014<br> Our Triumph</h5>
                    <p>Lorem ipsum dolor sit amet, consectetur<br>
                        adipisicing elit. Aliquid aut autem, beatae<br>
                        dicta dignissimos dolorem est ex excepturi facilis<br>
                        ipsam itaque nostrum nulla odio repellendus repudiandae
                        sed unde vero voluptatem?</p>
                </div>
                <div class="about-img right">
                    <img src="img/AboutUs/4.jpg" alt="">
                </div>
            </div>

            <a href="#" class="about_link"><br>Our<br>Story<br>Continues<br>...</a>
        </div>
    </div>
</div>

<div id="team" class="box">
    <div class="container">
        <h2 class="title">Our Amazing Team</h2>
        <p class="descriptions">Proin iaculis purus consequat sem cure.</p>

        <div class="team-wrap clearfix">
            <div class="team-item ">
                <img src="img/AboutUs/1.jpg" alt="">
                <h5 class="title">Kimberly Thompson</h5>
                <p class="category">Marketer</p>
                <div class="social">
                    <a href="#" class="ic-twitter"></a>
                    <a href="#" class="ic-facebook"></a>
                    <a href="#" class="ic-pn"></a>
                </div>
            </div>
            <div class="team-item ">
                <img src="img/AboutUs/1.jpg" alt="">
                <h5 class="title">Rico Massimo</h5>
                <p class="category">Coder</p>
                <div class="social">
                    <a href="#" class="ic-twitter"></a>
                    <a href="#" class="ic-facebook"></a>
                    <a href="#" class="ic-google"></a>
                </div>
            </div>
            <div class="team-item ">
                <img src="img/AboutUs/1.jpg" alt="">
                <h5 class="title">Uku Mason</h5>
                <p class="category">Graphic Designer</p>
                <div class="social">
                    <a href="#" class="ic-facebook"></a>
                    <a href="#" class="ic-pn"></a>
                </div>
            </div>
        </div>
        <p class="descriptions">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Consectetur dolorum earum et,<br>
            libero molestiae obcaecati perferendis recusandae velit veritatis! Fugit.</p>
    </div>
</div>

<div id="partners">
    <div class="container">
        <div class="partners-wrap clearfix">
            <div class="partners-logo envato">
                <img src="img/partners/envato.png" alt="">
            </div>

            <div class="partners-logo wordpress">
                <img src="img/partners/wordPress.png" alt="">
            </div>

            <div class="partners-logo tuts">
                <img src="img/partners/tuts+.png" alt="">
            </div>

            <div class="partners-logo microlancer">
                <img src="img/partners/microlancer.png" alt="">
            </div>
        </div>
    </div>
</div>

<div id="contact">
    <div class="container">
        <h2 class="title">Contact Us</h2>
        <p class="descriptions">Proin iaculis purus consequat sem cure.</p>

    <form action="#" id="message">
        <div class="clearfix">
            <div class="left">
                <input type="text" placeholder="your name *">
                <input type="text" placeholder="your e-mail *">
                <input type="text" placeholder="subject">
            </div>
            <div class="right">
                <textarea cols="0" rows="0" placeholder="your message *"></textarea>
            </div>
        </div>

        <input type="submit" value="send message">
    </form>

    </div>
</div>

<div id="footer">
    <div class="container">
        <div class="footer-wrap clearfix">
            <div class="copyright left text-left">
                <p>&copy; Copyright 2019 Dmitry Pinkovski</p>
            </div>
            <div class="social left">
                <a href="#" class="ic-twitter"></a>
                <a href="#" class="ic-facebook"></a>
                <a href="#" class="ic-pn"></a>
                <a href="#" class="ic-google"></a>
            </div>
        </div>
    </div>
</div>

</body>
</html>